src/use-link.hook.js

```diff
@@ -39,7 +39,7 @@ export function useLink({processAnchorElement = noop} = defaultOptions) {
   function insertLink(link, displayedText) {
     richTextContext.selectRangeFromBeforeBlur({usePreviousRange: true})
     const id = `rte-link-temp-id-${tempId++}`
-    performCommandWithValue(`<a href="${link}" id="${id}">${displayedText}</a>`)
+    performCommandWithValue(`<a href="${link}" id="${id}" target="_blank">${displayedText}</a>`)
```

Comment:
I think we should also add `rel="noopener noreferrer"` for reasons similar to explained here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a?target#Security_and_privacy_concerns

Reply:
👍 updated